tests/rptest/tests/admin_api_raft_metrics_test.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2022 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.services.admin import Admin
+from rptest.clients.types import TopicSpec
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.cluster import cluster
+from rptest.util import wait_until_result
+from rptest.util import wait_until
+import requests
+
+
+class AdminApiRaftMetricsTest(RedpandaTest):
+    topics = (TopicSpec(partition_count=1, replication_factor=3), )
+
+    def __init__(self, test_context):
+        extra_rp_conf = {
+            "enable_leader_balancer": False,
+            "partition_autobalancing_mode": "off",
+        }
+
+        super(AdminApiRaftMetricsTest,
+              self).__init__(test_context=test_context,
+                             num_brokers=3,
+                             extra_rp_conf=extra_rp_conf)
+
+        self.admin = Admin(self.redpanda)
+
+    @cluster(num_nodes=3)
+    def simple_test(self):
```

Comment:
Will add more interesting test when create request to get internal state for node